source/Calamari.Aws/Deployment/Conventions/DeleteCloudFormationStackConvention.cs

```diff
@@ -50,6 +41,7 @@ public override void Install(RunningDeployment deployment)
 
         private async Task InstallAsync(RunningDeployment deployment)
         {
+            deployment.Variables.Get("");
```

Comment:
What's this?

Reply:
Nice catch :) artifact from some testing